index.less

```diff
@@ -12,6 +10,7 @@
 @import "./filter-button";
 @import "./filter-menu-button";
 @import "./filter-menu";
+@import "./global";
```

Comment:
Sorry, just notice this. Can we move global and utility back to their original place? I believe they need to be first in the cascade. Possibly not an issue, but I'd rather not risk any regressions until I've had a chance to be sure.

Reply:
I did this in order to sort so that new modules appear in sorted order. How can we ensure this does not create regression? Any pointers?

Reply:
I think for now just move them back to the order they were in. In future we can look into whether it's safe to move them, but I don't want to hold up this PR.

Reply:
@raveenakothapally I think we are just waiting on this change, and then I can merge.

Reply:
@ianmcburnie This is fixed now and also code-gen ignores these legacy lines during the sort.